### PR TITLE
Commit Before Squash commits

### PR DESCRIPTION
**Pull request by rusakov66**
Based on Junit4

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>test_API</groupId>
+    <artifactId>request6</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20180130</version>
```
Comment (vboaghe) [inline pom.xml]: extract version to a property

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        </dependency>
+        <dependency>
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <version>4.0.0</version>
```
Comment (vboaghe) [inline pom.xml]: extract version to a property

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <version>1.3</version>
```
Comment (vboaghe) [inline pom.xml]: extract version to a property

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```
Comment (vboaghe) [inline pom.xml]: extract version to a property

**Comment by rusakov66** [comment]
I have reopen this PR for learning purpose only.

**Pull request closed without merging**